### PR TITLE
SWC-6869

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Depends on https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/999

File: src/main/java/org/sagebionetworks/web/client/GlobalApplicationStateImpl.java [hunk 1]
```diff
@@ -570,19 +570,32 @@ public void onPreviewNativeEvent(NativePreviewEvent event) {
...
+                  String href = targetElement.getAttribute("href");
+                  boolean handled = handleRelativePathClick(href);
+                  if (handled) {
+                    event.cancel();
+                  }
+                }
+              }
+            } catch (Exception e) {
+              synapseJSNIUtils.consoleError(e.getMessage());
+            }
```
Comment (nickgros) [inline src/main/java/org/sagebionetworks/web/client/GlobalApplicationStateImpl.java:598]: Does this work with the tabs in the `DataAccessManagement` place (ACT/AR reviewer dashboard), where the subpath is controlled by React Router?

**Review by nickgros — APPROVED** [review]
Had one question, but I see no reason to not try this without lots of automated e2e tests that we don't have today.

File: src/main/java/org/sagebionetworks/web/client/GlobalApplicationStateImpl.java (hunk 1 above)
Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/GlobalApplicationStateImpl.java:598]: Yes, verified that this works as expected, thanks for asking.
There is an uncaught exception whenever I switch tabs in this place, but that's true on production today as well (unexpected null).
Screen Shot 2024-06-12 at 2 33 34 PM

**Pull request merged**